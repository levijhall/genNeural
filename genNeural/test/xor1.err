Training stopped 3623 / 500000 (0.72%)
Avg. error = 0.001 < 0.001
[1.000 1.000 0.000 0.000 ] | [0.031 ] - [0.000 ] (0.001)
[1.000 1.000 0.000 1.000 ] | [0.968 ] - [1.000 ] (0.001)
[1.000 1.000 1.000 0.000 ] | [0.969 ] - [1.000 ] (0.001)
[1.000 1.000 1.000 1.000 ] | [0.032 ] - [0.000 ] (0.001)
