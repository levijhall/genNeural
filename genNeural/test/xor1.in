8
1 5 0.479
1 6 0.229
1 7 0.924
2 8 -41.306
3 5 0.175
3 6 0.293
3 7 0.950
4 5 0.607
4 6 0.195
4 7 0.506
5 8 2.926
6 8 -57.564
7 8 84.685
