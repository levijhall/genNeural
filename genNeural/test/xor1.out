8
1 5 0.795184
3 5 0.708419
4 5 0.632590
1 6 -1.306932
3 6 3.086094
4 6 3.111620
1 7 -3.263782
3 7 2.116849
4 7 2.138263
2 8 -1.547939
5 8 -0.545874
6 8 4.316726
7 8 -4.484589
